Assets/Scripts/Character/Enemies/BasicEnemy.cs

```diff
@@ -7,30 +7,30 @@
     public class BasicEnemy : Enemy
     {
         [SerializeField]
-        private float walkTime = 1;
```

Comment:
This class isn't meant to be extended, If you need these variables, make them again in your class.  While DRY normally applies, it doesn't in this case because we don't want changes in this script's logic to affect others.